src/strings.js

```diff
@@ -68,7 +72,23 @@ class Locales {
             is_a: " is a ",
             benetech: "Benetech",
             footer_description: "initiative supported by the U.S. Department of Education, Office of Special Education Programs (Cooperative Agreement #H327B100001). Opinions expressed herein are those of the authors and do not necessarily represent the position of the U.S. Department of Education. Poet™ is a trademark of Beneficent Technology, Inc. This website is copyright © 2012-2017, Beneficent Technology, Inc.",
-            loading: "Loading..."
+            loading: "Loading...",
+            share: "Share",
+            copy: " Copy",
+            close: " Close",
+            add_problem_title: "Add new Problem",
+            add_problem: " Add Problem",
+            add_problem_equation: "+",
+            save: "Save",
+            save_button: " Save",
+            edit_link_label: "Edit Url: ",
+            cancel: " Cancel",
+            new_problem: " New Problem",
+            add_problems: " Add problem(s) to set",
+            hash: "#",
```

Comment:
I don't think hash is needed to be in strings.js

Reply:
well technically it is locale dependant, at least I think so